private/bufpkg/bufmodule/bufmoduletesting/bufmoduletesting.go

```diff
@@ -124,10 +126,10 @@ var (
 	}
 	// TestDataWithDocumentation is the data that maps to TestDigestWithDocumentation.
 	//
-	// It includes a buf.md file.
+	// It includes a README.md file.
 	TestDataWithDocumentation = map[string][]byte{
-		TestFile1Path: []byte(`syntax="proto3";`),
-		"buf.md":      []byte(TestModuleDocumentation),
+		TestFile1Path:               []byte(`syntax="proto3";`),
+		TestModuleDocumentationPath: []byte(TestModuleDocumentation),
 	}
```

Comment:
Should we have two tests for this, one for `buf.md` and the other for the fallback paths? That way, we can also ensure the digest doesn't change in the old code path.

Reply:
+1

Reply:
Returning the existing test to use `buf.md`, and added another test for `README.md`.